src/test/java/picard/illumina/parser/TileMetricsUtilTest.java

```diff
@@ -8,67 +8,90 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Random;
 import java.util.stream.IntStream;
 
 public class TileMetricsUtilTest {
     @Test
     public void testFindTileMetrics() throws IOException {
-        Path baseDir = createTestDirs(0, true);
-        File tileMetricsFile = TileMetricsUtil.renderTileMetricsFileFromBasecallingDirectory(
+        List<Integer> populatedCycleDirs = new ArrayList<>();
```

Comment:
It looks like this collection is never modified and is always empty. If so, it would be more efficient to use `Collections.emptyList()` instead of allocating a new list.